src/main/java/Duck.java

```diff
@@ -0,0 +1,164 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duck {
+    static String divider = "____________________________________________________________";
+    static String logo = " _____             _    \n"
+            + "|  __  \\_   _  ___| | __\n"
+            + "| |  | | | | |/ __| |/ /   _\n"
+            + "| |__| | |_| | (__|   <  <(.)___\n"
+            + "|_____/ \\__,_|\\___|_|\\_\\  (____/\n";
+    static private ArrayList<Task> taskList = new ArrayList<Task>();
+    static private int taskCounter = 0;
+
+    public static void main(String[] args) {
+        printWelcomeMessage();
+        while (true) {
```

Comment:
i love the extraction of this code into its own function, great way to keep it simple